Lib/Src/CUartCom.cpp

```diff
@@ -286,16 +255,12 @@ bool CUartCom::isDataAvailable()
 
 /**
  * @brief Get and delete the first element in the RX queue
- * @return Data as string
+ * @return byte of data
  */
-etl::string<MAX_STRING_SIZE> CUartCom::getData()
+uint8_t CUartCom::getData()
```

Comment:
Why? I fail to see the point of returning a single byte.

Reply:
If I have a JSON parser, maybe I can get a stream of data that does not include a '\0' or '\n' character, but it still has the '}' character. 
If we work with strings, that command will never be recognised, but if we let each parser process the data this could work. 
As I said earlier, I don't think UART should process anything, just store all the incoming data.